receiver/azureblobrehydrationreceiver/config.go

```diff
@@ -25,6 +25,9 @@ import (
 
 // Config is the configuration for the azure blob rehydration receiver
 type Config struct {
+	// BatchSize is the number of blobs to process entering the pipeline in a single batch. (default 30)
+	BatchSize int `mapstructure:"batch_size"`
```

Comment:
We might want to comment about concurrency here, and mention that each blob will be responsible for a connection to azure. Funny enough, I ran out of file handles (limited to 1024 by default) when configuring this to 10,000. It was working for the most part.

```suggestion
	// BatchSize is the number of blobs to process entering the pipeline in a single batch. (default 30)
	BatchSize int `mapstructure:"batch_size"`
```

Reply:
It is mentioned in the readme, but yep I can update it here as well!